store/src/java/com/zimbra/cs/filter/JsieveConfigMapHandler.java

```diff
@@ -69,14 +69,7 @@
             ZimbraLog.filter.debug("Zimbra 'notify' is loaded");
         }
 
-        boolean isRejectSupported = true;
-        try {
-            isRejectSupported = Provisioning.getInstance().getConfig().getBooleanAttr(
-                Provisioning.A_zimbraSieveRejectEnabled, true);
-        } catch (ServiceException e) {
-            // 'reject' action is supported as default
-        }
-        if (isRejectSupported) {
+        if (isZimbraSieveRejectEnabled()) {
             mCommandMap.put("reject", com.zimbra.cs.filter.jsieve.Reject.class.getName());
```

Comment:
You should use inheritance instead of adding conditional classes to map. 
That will remove server dependency.